lib/dor/services/metadata_service.rb

```diff
@@ -62,20 +63,25 @@ def label_for(identifier)
 
       def handler_for(prefix)
         handler = handlers[prefix.to_sym]
-        raise MetadataError, "Unkown metadata prefix: #{prefix}" if handler.nil?
+        raise MetadataError, "Unknown metadata prefix: #{prefix}" if handler.nil?
 
         handler
       end
 
       private
 
       def handlers
-        @handlers ||= {}
+        @handlers ||= {}.tap do |handlers|
+          # There's only one. If additional handlers are added, will need to be registered here.
+          register(CatalogHandler.new, handlers)
+        end
+      end
+
+      def register(handler, handlers)
```

Comment:
would you mind renaming `handlers` so as not to confuse it with the method of the same name.

Reply:
So at least on line 73, rubocop insists:
```